tests/test_manifest.py

```diff
@@ -78,3 +78,18 @@ def test_include_section_is_required():
         validate_manifest_definition(manifest_definition)
 
     assert len(str(e))
+
+
+def test_filter_manifest_definition():
+
+    manifest_definition = {'package': {'output': './build'},
+                           'functions': {'sample-1': {'includes': ['./']},
+                                         'sample-2': {'includes': ['./']},
+                                         'another': {'requirements': './requirements/dev.txt',
+                                                     'include': ['./']}}}
+    expected_manifest_definition = {'package': {'output': './build'},
+                                    'functions': {'sample-1': {'includes': ['./']},
+                                                  'sample-2': {'includes': ['./']}}}
+
+    actual_manifest_definition = filter_manifest_definition(manifest_definition, 'sample')
+    assert actual_manifest_definition == expected_manifest_definition
```

Comment:
add a new test in which the keyword appears somewhere in the middle of the function name.